--- origsrc/java-atk-wrapper-0.30.1/jni/src/AtkWrapper.c	2010-05-28 03:25:04.000000000 -0500
+++ src/java-atk-wrapper-0.30.1/jni/src/AtkWrapper.c	2010-06-20 15:10:30.192370300 -0500
@@ -172,8 +172,8 @@ JNIEXPORT jboolean JNICALL Java_org_GNOM
 		printf("GTK_PATH=%s\n", gtk_module_path);
 	}
 
-	const gchar* atk_bridge_file = g_strconcat(gtk_module_path,
-			"/modules/libatk-bridge.so", NULL);
+	gtk_module_path = g_strconcat(gtk_module_path, "/modules", NULL);
+	const gchar* atk_bridge_file = g_module_build_path(gtk_module_path, "atk-bridge");
 
 	if (jaw_debug) {
 		printf("We are going to load %s\n", atk_bridge_file);
--- origsrc/java-atk-wrapper-0.30.1/jni/src/Makefile.am	2010-05-28 03:25:35.000000000 -0500
+++ src/java-atk-wrapper-0.30.1/jni/src/Makefile.am	2010-06-20 15:06:51.872883100 -0500
@@ -26,7 +26,7 @@ libatk_wrapper_la_CPPFLAGS = $(JNI_INCLU
 			     $(JAW_CFLAGS)\
 			     -DATK_BRIDGE_LIB_PATH=\"$(ATK_BRIDGE_LIB_PATH)\"
 
-libatk_wrapper_la_LDFLAGS = -version-info $(LIB_ATK_WRAPPER_LT_VERSION)
+libatk_wrapper_la_LDFLAGS = -avoid-version -no-undefined
 
 libatk_wrapper_la_LIBADD = $(JAW_LIBS)
 
